### PR TITLE
Add default labels to issues

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Add default labels to issues to mirror simorgh [issue templates](https://github.com/bbc/simorgh/tree/latest/.github/ISSUE_TEMPLATE)

**Code changes:**

- Add default labels
- Remove prompts
- Bump psammead and add changelog update

**Testing notes:**

- No testing required, github admin impact only

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Tests added for new features~
- [ ] Test engineer approval

### CODE REVIEW

**Review by DenisHdz — DISMISSED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "1.1.4",
+  "version": "1.2.0",
```
Comment (sareh) [inline package.json]:
```suggestion
  "version": "1.1.5",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "1.1.4",
+  "version": "1.2.0",
```
Comment (sareh) [inline package-lock.json]:
```suggestion
  "version": "1.1.5",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0   | [PR#484](https://github.com/BBC/psammead/pull/484) Add default labels to github issue templates |
```
Comment (sareh) [inline CHANGELOG.md]:
```suggestion
| 1.1.5   | [PR#484](https://github.com/BBC/psammead/pull/484) Add default labels to GitHub issue templates |
```

**Review by sareh — CHANGES_REQUESTED** [review]

**Review by sareh — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]
Noiceeeee

**Comment by david-boydell** [comment]
Just a template amend, no testing required.

**Pull request merged**